app/main.py

```diff
@@ -2,0 +1,24 @@
+import os
+import shutil
+
+
+def move_file(command: str) -> None:
+    data = command.split(" ")
+    if data[0] == "mv" and len(data) == 3:
+        source_file = data[1]
+        destination = data[2]
+
+        if not os.path.exists(source_file):
+            raise FileNotFoundError(f"File {source_file} does not exist.")
+
+        if destination.endswith("/"):
```

Comment:
The check `if destination.endswith('/'):` is used to determine if the destination is a directory. This is a good approach, but it assumes that the user will always end directory paths with a '/'. Consider using `os.path.isdir(destination)` to check if the destination is a directory more reliably.